cilium-dbg/cmd/bpf_frag_list.go

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/maps/fragmap"
+)
+
+const (
+	fragKeyTitle   = "PROTO ID SOURCE_ADDR --> DEST_ADDR"
+	fragValueTitle = "SOURCE_PORT DST_PORT"
```

Comment:
It would look cleaner to me to have separate columns and to populate them independently, so that the content is correctly aligned with the headers, maybe?

Reply:
OK

Reply:
Updated.